integrations/azure-devops/azure_devops/client/azure_devops_client.py

```diff
@@ -258,6 +258,49 @@ async def get_repository(self, repository_id: str) -> dict[Any, Any]:
         repository_data = response.json()
         return repository_data
 
+    async def get_columns(self) -> AsyncGenerator[list[dict[str, Any]], None]:
+        async for boards in self.get_boards_in_organization():
+            for board in boards:
+                yield [
+                    {
+                        **column,
+                        "__board_id": board.get("id"),
+                        "__project": board.get("__project"),
+                    }
+                    for column in board.get("columns", [])
```

Comment:
I don't see how this will fit in since the boards API return only name, id and url. How do we get the columns to be returned as part of the boards?

Reply:
There is another function that enriches the data on line 273